src/controllers/resize-controller.stories.js

```diff
@@ -0,0 +1,328 @@
+import '../components/cc-expand/cc-expand.js';
+import { html, css, LitElement } from 'lit';
+import { enhanceStoriesNames } from '../stories/lib/story-names.js';
+import { ResizeController } from './resize-controller.js';
+import docsPage from './resize-controller.md';
+
+class DemoContainer extends LitElement {
+  static get properties () {
+    return {
+      breakpoints: { type: Object },
+    };
+  }
+
+  constructor () {
+    super();
+
+    this._teams = [
+      {
+        name: 'Hubert',
+        team: 'Cat 😺',
+      },
+      {
+        name: 'Mathieu',
+        team: 'Cat 😺',
+      },
+      {
+        name: 'Florian',
+        team: 'Dog 🐶',
+      },
+      {
+        name: 'Robert',
+        team: 'Dog 🐶',
+      },
+      {
+        name: 'Pierre',
+        team: 'NC 🏳️',
```

Comment:
I have :frog: in my pond ;)

Reply:
Frogs are cool too, I'll add it to the story :joy: